tsdb/points_test.go

```diff
@@ -594,7 +662,7 @@ func TestParsePointEscapedStringsAndCommas(t *testing.T) {
 }
 
 func TestParsePointWithStringWithEquals(t *testing.T) {
-	test(t, `cpu,host=serverA,region=us-east str="foo=bar",value=1.0, 1000000000`,
```

Comment:
This test actually shouldn't have passed but was before.  There is a comma at the end of field section.